definitions/divorce/json/AuthorisationCaseEvent.json

```diff
@@ -5072,7 +5072,7 @@
     "CaseTypeID": "DIVORCE",
     "CaseEventID": "IssueAOSOffline_Respondent_fromAosAwaiting",
     "UserRole": "caseworker-divorce-courtadmin_beta",
-    "CRUD": "R"
+    "CRUD": "CRUD"
```

Comment:
@jvictorcf can you please confirm this is what we need to "toggle on" this feature? :)

Reply:
This will enable it from one of the states (AOS Awaiting). We probably need to do it for all other states as well.

Are we releasing it as we develop the stories or are going to release them all at once?

Reply:
@yarpo ^

Reply:
I think we might not want to release that yet until everything is ready. We can upload it on AAT to test things for now.

Reply:
@Dizrupto  Agreed